helm_deploy/prisoner-content-hub-frontend/values.local.yaml

```diff
@@ -7,7 +7,6 @@ application:
     feedbackServiceName: hub-elasticsearch-elasticsearch-coordinating-only
     feedbackEndpoint: /prod-feedback/_doc
     analyticsSiteId: UA-152065860-4
-    enableRedisCache: false
```

Comment:
we don't have redis in any environment - this would only be used to share an api token between 3 clients. Removing this from helm config and will look at removing entirely